README.md

```diff
@@ -21,27 +16,21 @@ The Notify PHP Client can be installed with [Composer](https://getcomposer.org/)
 composer require php-http/guzzle6-adapter alphagov/notifications-php-client
 ```
 
-## Usage
+## Getting started
 
-Assuming you've installed the package via Composer, the Notify PHP Client will be available via the autoloader.
+Assuming you’ve installed the package via Composer, the Notify PHP Client will be available via the autoloader.
 
 Create a (Guzzle v6 based) instance of the Client using:
 
 ```php
 $notifyClient = new \Alphagov\Notifications\Client([
-    'serviceId'     => '{your service id}',
-    'apiKey'        => '{your api key}',
+    'apiKey' => '{your api key}',
```

Comment:
i call it that after this old sublime text plugin i used to use until my old colleagues shouted at me https://github.com/khiltd/Abacus